components/match_ticker/commons/match_ticker_display_components_new.lua

```diff
@@ -0,0 +1,382 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:MatchTicker/DisplayComponents/New
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+-- Holds DisplayComponents for the MatchTicker module
+
+local Abbreviation = require('Module:Abbreviation')
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Countdown = require('Module:Countdown')
+local DateExt = require('Module:Date/Ext')
+local String = require('Module:StringUtils')
+local LeagueIcon = require('Module:LeagueIcon')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+local Timezone = require('Module:Timezone')
+local StreamLinks = require('Module:Links/Stream')
+
+local HighlightConditions = Lua.import('Module:HighlightConditions')
+
+local OpponentLibraries = require('Module:OpponentLibraries')
+local Opponent = OpponentLibraries.Opponent
+local OpponentDisplay = OpponentLibraries.OpponentDisplay
+
+local VS = 'VS'
+local SCORE_STATUS = 'S'
+local CURRENT_PAGE = mw.title.getCurrentTitle().text
+local HIGHLIGHT_CLASS = 'tournament-highlighted-bg'
+local TOURNAMENT_DEFAULT_ICON = 'Generic_Tournament_icon.png'
+local NOW = os.date('%Y-%m-%d %H:%M', os.time(os.date('!*t') --[[@as osdateparam]]))
+
+---Display class for the header of a match ticker
+---@class MatchTickerHeader
```

Comment:
fyi defining the class with the same name as in the old version will cause annotation warnings ;)